core/src/apps/nem/transfer/serialize.py

```diff
@@ -19,13 +19,17 @@
     write_uint64_le,
 )
 
+if False:
+    from trezor.crypto import bip32
+    from trezor.utils import Writer
+
 
 def serialize_transfer(
     common: NEMTransactionCommon,
     transfer: NEMTransfer,
     public_key: bytes,
-    payload: bytes = None,
-    encrypted: bool = False,
+    payload: bytes | None = None,
+    is_encrypted: bool = False,
```

Comment:
These arguments should not have default values. `payload` probably shouldn't be `bytes | None` but just `bytes` ?

Reply:
Right, this function is used only once and all parameters are supplied. I will remove the default values

Reply:
Removed in 677fa35